src/sass/components/_footer.scss

```diff
@@ -13,25 +12,24 @@
     right: 0;
     transform: translateY(-89px);
     height: 1px;
-    background-color: $text;
+    background-color: $gray;
   }
 
   &__header {
-    @include font-mixin('Titan One', 400, 18px, 30px, 0.04em, $brand-pink);
+    @include font-mixin('Titan One', 400, 18px, 21px, 0em, $brand-pink);
     flex-wrap: wrap;
```

Comment:
w tej klasie nie masz nawet `display: flex;` więc to Ci nie działa, można usunąć